content/docs/docs-nav.yaml

```diff
@@ -43,6 +39,11 @@ nav:
     - Travis CI: '/docs/integrations/travis-ci/'
     - Tech radar: '/docs/configuration/tech-radar/'
     - Cortex: '/docs/integrations/cortex'
+  - Integrations:
+    - Google Cloud Platform: '/docs/integrations/gcp/'
```

Comment:
Move this to be second to last to order alphabetically.

Reply:
Done